nix/ext/wrappers/default.nix

```diff
@@ -50,10 +50,46 @@ buildPgrxExtension_0_12_6 rec {
   CARGO_BUILD_JOBS = "2";
   CARGO="${cargo}/bin/cargo";
   
+  #CARGO_NET_GIT_FETCH_WITH_CLI = "true";
   cargoLock = {
     lockFile = "${src}/Cargo.lock";
     allowBuiltinFetchGit = true;
   };
+
+ preConfigure = ''
+    cd wrappers
+    
+    # update the clickhouse-rs dependency
+    # append the branch name to the git URL to help cargo locate the commit
+    # while maintaining the rev for reproducibility
+    awk -i inplace '
+    /\[dependencies.clickhouse-rs\]/ {
+      print
+      getline
+      if ($0 ~ /git =/) {
+        print "git = \"https://github.com/suharev7/clickhouse-rs/async-await\""
+      } else {
+        print
+      }
+      while ($0 !~ /^\[/ && NF > 0) {
+        getline
+        if ($0 ~ /rev =/) print
+        if ($0 ~ /^\[/) print
+      }
+      next
+    }
+    { print }
+    ' Cargo.toml
+    
+    # Verify the file is still valid TOML, break build with this error
+    # if it is not
+    if ! cargo verify-project 2>/dev/null; then
+      echo "Failed to maintain valid TOML syntax"
+      exit 1
+    fi
+    
+    cd ..
+  '';
   
   buildAndTestSubdir = "wrappers";
   buildFeatures = [
```

Comment:
We can replace this with `all_fdws` so that they get pulled in automatically as they're added, rather than us ensuring that newly introduced are covered - https://github.com/supabase/wrappers/blob/main/wrappers/Cargo.toml#L157-L160

Reply:
@pcnc if it's ok with you I will follow up with a PR to do this rather than on this one

Reply:
Yup, sounds good!